app/src/main/java/io/gnosis/safe/ui/safe/add/AddSafeNameViewModel.kt

```diff
@@ -32,6 +32,7 @@ class AddSafeNameViewModel
                 updateState { AddSafeNameState(ViewAction.ShowError(it)) }
             }.onSuccess {
                 updateState { AddSafeNameState(ViewAction.CloseScreen) }
+                tracker.setNumSafes(safeRepository.getSafes().count())
```

Comment:
With a singleton implementation I suggested then we could
```suggestion
                Tracker.get().setNumSafes(safeRepository.getSafes().count())
```

and get rid of the `tracker` in the constructor